install_config/router/proxy_protocol.adoc

```diff
@@ -111,11 +106,14 @@ $ aws elb create-load-balancer --load-balancer-name "$lb" \
     "DNSName": "infra-lb-2006263232.us-east-1.elb.amazonaws.com"
 }
 ----
-====
-+
-3. Register your router instance or instances with the ELB:
 +
+[NOTE]
 ====
+You must configure all listeners to use the TCP protocol, not the HTTP protocol.
+====
```

Comment:
I can see the note box above came from before the step, but I'm not sure being after the step is best. Can I suggest:

* Placing the note box above the example with a rewrite to something like "Note that the following example configures all listeners to use the TCP protocol and not the HTTP protocol"

or

* Putting a footnote into the example with something like "Using Protocol=TCP configures all listeners to use the TCP protocol and not the HTTP protocol, which is necessary for these steps to work"

WDYT?